sdk/documentintelligence/azure-ai-documentintelligence/src/samples/README.md

```diff
@@ -4,8 +4,7 @@ languages:
   - java
 products:
   - azure
-  - azure-cognitive-services
```

Comment:
https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main

https://dev.azure.com/azure-sdk/public/_build/results?buildId=3258134&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=ac8f4042-9b76-5db4-27b1-2a4abaa9bb3c&l=18
?

Reply:
https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Test-SampleMetadata.ps1#L198

You will need to add the new slug in the link above.

Reply:
https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/common_engsys.md#workflow